src/script/cpp_api/s_base.cpp

```diff
@@ -266,6 +266,11 @@ std::string ScriptApiBase::getCurrentModName(lua_State *L)
 void ScriptApiBase::loadMod(const std::string &script_path,
 		const std::string &mod_name)
 {
+	if (!fs::IsFile(script_path)) {
+		errorstream << "Mod error: Failed to load mod \"" << mod_name
```

Comment:
Unwanted behavioral change IMO.

Minetest *should* error (rather than just logging an easy to miss error message) when a "mod" is incomplete to force user action (disabling or uninstalling the "mod").

Reply:
Agreed. Made exception throwing instead.

However, due to multithreading, Minetest has very small chance to crash on server closing. But, for me, it happens only one time after building debug version and doesn't repeat until rebuild. Maybe it happens only one time after `s_base.cpp` rebuild and doesn't affect release version, but I give stack trace here anyway:

Exception: Access violation
>	minetest.exe!std::vector<NodeResolver *,std::allocator<NodeResolver *>>::size() Line 1731	C++
 	minetest.exe!NodeDefManager::cancelNodeResolveCallback(NodeResolver * nr) Line 1622	C++
 	minetest.exe!NodeResolver::~NodeResolver() Line 1770	C++
 	[External code]	
 	minetest.exe!ObjDefManager::~ObjDefManager() Line 35	C++
 	minetest.exe!BiomeManager::~BiomeManager() Line 214	C++
 	[External code]	
 	minetest.exe!EmergeManager::~EmergeManager() Line 153	C++
 	[External code]	
 	minetest.exe!Server::~Server() Строка 414	C++
 	[External code]	
 	minetest.exe!Game::shutdown::__l2::<lambda>() Line 1294	C++
 	[External code]	
 	minetest.exe!LambdaThread::run() Строка 41	C++
 	minetest.exe!Thread::threadProc(Thread * thr) Line 194	C++
 	[External code]